dex/src/main/scala/com/wavesplatform/dex/tool/ConfigChecker.scala

```diff
@@ -27,20 +27,22 @@ import scala.util.Try
 
 object ConfigChecker extends ConfigWriters {
 
-  val skippedPaths = Seq(
-    "events-queue.kafka.consumer.client",
-    "events-queue.kafka.producer.client"
-  )
+  private val skippedPathsDelimiter = ";"
```

Comment:
We don't need this and split below, it should work :)

Reply:
Yes) Done